helm/crds/sagemaker.services.k8s.aws_trainingjobs.yaml

```diff
@@ -516,6 +516,32 @@ spec:
               failureReason:
                 description: If the training job failed, the reason it failed.
                 type: string
+              modelArtifacts:
+                description: Information about the Amazon S3 location that is configured
+                  for storing model artifacts.
+                properties:
+                  s3ModelArtifacts:
+                    type: string
+                type: object
+              profilerRuleEvaluationStatuses:
```

Comment:
why is this a part of this PR, SDK version hasn't changed

Reply:
Was auto-generated via code-gen

Reply:
hmm, not sure why it is on the PR but it should be there  in the helm either way, so ok.

Reply:
I was using an older version of code gen, this change is in the latest codegen